internal/transaction/meta.go

```diff
@@ -0,0 +1,38 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package transaction
+
+import "cqfn.org/degitx/locators"
+
+// Vote of RM
+type Vote uint8
+
+const (
+	// VotePrepared means that RM was prepared to commit
+	VotePrepared Vote = iota
+	// VoteAborted means that RM failed to prepare
+	VoteAborted
+	// VoteUnkown means that RM was not decided yet
+	VoteUnkown
+)
+
+// Scope of the transaction
+type Scope struct {
+	// Acceptors for this transaction
+	Acceptors []*locators.Node
```

Comment:
Probably it's better to have only nodes in transaction's scope? While acceptors are virtual and are managed by a node. So they should be encapsulated, and node should parse a request to acceptor and redirect to proper one internally.

Reply:
@stepanvalyavskiy I didn't get this coment. What did you mean by "have only nodes in transaction's scope"? Acceptors belongs to the transaction scope right now, as you have said.

Reply:
Forget it, I misunderstood the interface `Manager`